Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/auditrepository/hibernate/AuditRepositoryRecord.java

```diff
@@ -147,4 +139,18 @@ public void setTransactionId(String transactionId) {
         this.transactionId = transactionId;
```

Comment:
As Tabassum mentioned tarnsactionId is removed from schema and this can be removed.